projects/hslayers/components/draw/draw-panel/draw-panel.component.ts

```diff
@@ -1,19 +1,42 @@
-import {Component} from '@angular/core';
+import {Component, input} from '@angular/core';
 
 import {Circle, Fill, Stroke, Style} from 'ol/style';
 
+import {AsyncPipe, NgClass, NgForOf} from '@angular/common';
+import {FormsModule} from '@angular/forms';
 import {HsDrawService} from 'hslayers-ng/services/draw';
-import {HsLanguageService} from 'hslayers-ng/services/language';
+import {
+  HsLanguageService,
+  TranslateCustomPipe,
+} from 'hslayers-ng/services/language';
 import {HsLayerUtilsService} from 'hslayers-ng/services/utils';
 import {HsLayoutService} from 'hslayers-ng/services/layout';
+import {NgbDropdownModule} from '@ng-bootstrap/ng-bootstrap';
 import {getTitle} from 'hslayers-ng/common/extensions';
 
+interface DrawToolDefinition {
+  type: string;
+  icon: string;
+  title: string;
+}
+
 @Component({
   selector: 'hs-draw-panel',
   templateUrl: './draw-panel.component.html',
   styleUrls: ['./draw-panel.component.scss'],
+  standalone: true,
+  imports: [
+    AsyncPipe,
+    TranslateCustomPipe,
+    NgClass,
+    NgForOf,
```

Comment:
Shouldn't be necessary when only control flow blocks are used (no `*ngFor`).

Reply:
[9335610](https://github.com/hslayers/hslayers-ng/pull/5317/commits/9335610ba3ac7ace4a16b9fad6b214eb3541acad)